CHANGELOG.md

```diff
@@ -37,7 +37,27 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
-* nothing
+### Features
+
+* Add the (empty) `saffron-rc2` upgrade [#1699](https://github.com/provenance-io/provenance/issues/1699).
```

Comment:
This should probably be updated as it's no longer empty.

Reply:
Fixed. :D